### DISCUSSION TITLE
12/05 신명희 블로그 3회차

### DISCUSSION THREAD

**Discussion by mh0223** [Orm-camp/blog-feedback#45] [동글동글 3기 피드백]
<a href="https://velog.io/@mh0223/JS-%EC%8B%A4%ED%96%89-%EC%BB%A8%ED%85%8D%EC%8A%A4%ED%8A%B8-1">링크 </a>

**Comment by kimjigyun** [comment]
세상에나 .. 전 아직 명희님 글을 읽을 레벨의 독자가 아닌듯한 어려움
어서 빨리 발전하겠습니다.

**Reply by mh0223** [reply]
앗..!!!! 아닙니다,, 제가 책 읽고 정리한 내용이다보니 용어를 많이 사용하게 된 거 같아요..!! 담 포스팅엔 더 쉽게 이해할 수 있도록 그림 같은 거도 추가할 예정인데 그 때도 피드백 남겨주세요!!

**Comment by ghost** [comment]
문단으로 나누고 요약도 간략하게 되어 있어서 읽기 편했습니다
